filesystem/iso9660.ksy

```diff
@@ -778,6 +809,12 @@ types:
           - id: file_structure_version
             doc-ref: ecma-119 8.4.31
             type: s1
+        instances:
+          path_table:
+            io: _root._io
```

Comment:
Are you sure it should positioned manually to that offset?

Reply:
Yes I am sure, I have reached the end of the current stream.
The data offset is documented as a sector on the disk